view/theme/frio/php/default.php

```diff
@@ -37,14 +37,14 @@
 		if (is_null($uid)) {
 			$uid = get_theme_uid();
 		}
-		$schema = get_pconfig($uid, 'frio', 'schema');
+		$schema = Friendica\Core\PConfig::get($uid, 'frio', 'schema');
```

Comment:
What was your motivation not to use `use` at the to of this file?

Reply:
Just that there were already no use statments and Friendica\Core\System::baseUrl() was used elsewhere in file, so I just mirrored that usage. I can certainly change it if you like.

Reply:
My own policy is to use the fully qualified class name if it's only used once, and use a `use` statement for two or more uses. There's no objective benefit to use either, other than visual habits maybe? I personally would change it in this file because there are multiple `PConfig` calls, but not in the other file where there's only one use.